docs/src/tools.rst

```diff
@@ -85,6 +85,33 @@ Below are the actions you can take on your CDK app:
    If one of cdk.json or ~/.cdk.json exists, options specified there will be used
    as defaults. Settings in cdk.json take precedence.
 
+.. _using-a-proxy:
+
+Using a proxy
+~~~~~~~~~~~~~
+
+If you use a proxy to connect to the Internet, configure it by setting either the `HTTPS_PROXY` or
```

Comment:
Can we add a one line note in “cdk —help” about this?

Reply:
I suppose we can make it a real command-line argument as well.